rtloader/three/three.cpp

```diff
@@ -110,7 +110,10 @@ bool Three::init()
     PyImport_AppendInittab(KUBEUTIL_MODULE_NAME, PyInit_kubeutil);
     PyImport_AppendInittab(CONTAINERS_MODULE_NAME, PyInit_containers);
 
-    Py_Initialize();
+    // force initialize siginterrupt with signal in python so it can be overwritten by the agent
+    // This only effects the windows builds as linux already has the sigint handler initialized
+    // and thus python will ignore it
+    Py_InitializeEx(1);
```

Comment:
This doesn't seem to actually force anything, since this behavior is what we have today by the default:

> This function works like [Py_Initialize()](https://docs.python.org/3/c-api/init.html#c.Py_Initialize) if initsigs is 1.

Reply:
I was just making it more explicit, but can go back the the current implantation.